service/controller/key/generate/generate_amis.go

```diff
@@ -104,45 +102,32 @@ func scrapeVersionAMIs(source io.Reader) (map[string]string, error) {
 }
 
 func main() {
```

Comment:
Did we add some docs on how we do this? Something maybe in this operators readme would be good.

Reply:
Good call. I added some. Let me know what you think.